app/models/preference.rb

```diff
@@ -24,4 +24,11 @@ class Preference < ApplicationRecord
   validates :name, presence: true
   validates :description, presence: true
   validates :restriction, inclusion: { in: [true, false] }
+  validate :validate_preferences_number, on: :create
```

Comment:
I think is redundant, wdyt?

```suggestion
  validate :preferences_number, on: :create
```